weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx

```diff
@@ -16,6 +19,20 @@ const SUMMARY_FIELDS_EXCLUDED_FROM_GENERAL_RENDER = [
   'weave',
 ];
 
+const StorageSizeDisplay: React.FC<{
+  size: number;
+}> = ({size}) => (
+  <span className="flex items-center gap-2">
+    {convertBytes(size)}
+    <Tooltip
+      content="The size does not include referenced objects, for example, images or audio blob storage."
+      trigger={
+        <Button icon="info" variant="ghost" size="small" active={false} />
+      }
+    />
+  </span>
+);
+
```

Comment:
Interesting that you are using class-based component here. I guess it's to comply with the rest of this file.